### PR TITLE
Like condition

### PR DESCRIPTION
**Pull request by freidonuten**
solves #18

### CODE REVIEW

File: src/main/java/eu/qwsome/sql/condition/Like.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+public class Like extends BiCondition {
+
+    Like(ValueHolder first, ValueHolder second) {
+        super(first, second);
+    }
+
+    @Override
+    String getOperator() {
+        return " like ";
```
Comment (sgflt) [inline src/main/java/eu/qwsome/sql/condition/Like.java]: Please use upper case for SQL keywords.

File: src/main/java/eu/qwsome/sql/condition/NotLike.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+public class NotLike extends BiCondition {
+
+    NotLike(ValueHolder first, ValueHolder second) {
+        super(first, second);
+    }
+
+    @Override
+    String getOperator() {
+        return " not like ";
```
Comment (sgflt) [inline src/main/java/eu/qwsome/sql/condition/NotLike.java]: Please use upper case for SQL keywords.

**Review by sgflt — CHANGES_REQUESTED** [review]

File: src/main/java/eu/qwsome/sql/condition/NotLike.java (hunk 2 above)
Comment (freidonuten) [reply src/main/java/eu/qwsome/sql/condition/NotLike.java]: resolved in b0988c8

File: src/main/java/eu/qwsome/sql/condition/Like.java (hunk 1 above)
Comment (freidonuten) [reply src/main/java/eu/qwsome/sql/condition/Like.java]: resolved in b0988c8

**Review by sgflt — APPROVED** [review]

**Pull request merged**